### ISSUE TITLE
New font size for more flexible layout

### ISSUE THREAD

**Issue by Mariefhrmnn** [telus/tds-core#939]
## Description

TDS library proposes 4 heading font sizes: H1-H2-H3-H4 (large size), but it's not enough to create a layout with a good reading level for the user. 

## Recommendation

To have more flexibility, I propose to improve this case developing one intermediate size between H2 and H3 

## Designs

**Current design:**
![screen shot 2019-02-14 at 11 23 18 am](https://user-images.githubusercontent.com/44266628/52803165-7aa9aa80-304f-11e9-98ab-67f3b2f8e395.png)
![screen shot 2019-02-14 at 11 27 00 am](https://user-images.githubusercontent.com/44266628/52803180-85643f80-304f-11e9-8f15-8911b96e7bf1.png)

**Improved design:** 
![screen shot 2019-02-14 at 11 23 06 am](https://user-images.githubusercontent.com/44266628/52803169-7da49b00-304f-11e9-9cc6-144244d44f17.png)
![screen shot 2019-02-14 at 11 27 16 am](https://user-images.githubusercontent.com/44266628/52803183-885f3000-304f-11e9-83d1-3e93fb040cd7.png)

## Meta

- TDS component: 1.1.6
- Willing to develop solution: Yes
- Has workaround: No
- High impact: No
- Designer/Developer Pair: Marie Fuhrmann / @Berjesty and @jessegrunert / @invalidred / @Jefey-Chang

**Comment by theetrain** [comment]
Hi Marie, thanks for starting the conversation. Please let us know what design leads think about this suggestion here.

**Comment by marcod1419** [comment]
Hi Marie! We will be evaluating adding this functionality in when we improve our Typography components. Thanks for reporting!

**Comment by Mariefhrmnn** [comment]
Hey @marcod1419,
Do you know the approximate date? Because It will be great to have that update for this summer!

**Comment by marcod1419** [comment]
Hi @Mariefhrmnn, this is planned on our roadmap for Q3.

**Comment by Mariefhrmnn** [comment]
Hey TDS team,
I met an other font issue when I created the Billing page layout. 
If I use the current font sizes that we have in our library, I have a bad reading level (first screenshot) between 'New charge', 'Alex Lee (416) 545-9966' and 'Recurring monthly plan charges'.
But when I change the size of **expand collapse heading** it's better.

Also, I notified that on many websites, the text of drop dow + body copy for the desktop version is never bigger than 16/18 px. 

@Christina-Lo / @serenechen-UX-Billing [](url)

![BAD Screen Shot 2019-08-21 at 3 45 05 PM](https://user-images.githubusercontent.com/44266628/63464194-b96c5300-c42c-11e9-9f17-9d4b3da7bc95.png)
![GOOD Screen Shot 2019-08-21 at 3 45 20 PM](https://user-images.githubusercontent.com/44266628/63464195-ba04e980-c42c-11e9-9742-51700c70d2ac.png)
